setup.py

```diff
@@ -35,6 +35,7 @@
             "pre-commit",
             "black",
             "flake8",
+            "flake8-docstrings",
```

Comment:
I don't understand how `flake8-docstrings` has been configured above without this dev dependency.

Reply:
Ok so pre-commit's libraries / dependencies are separate from pip. So in the pre-commit config above, the `repo` field lists the place where pre-commit would pull the code from and it handles how it packages / manages versions of libraries which do checks.

If we want python / python to be the only thing that handles dependencies we could make these checks run against the local repository. We would move the checks under the `repo: local` grouping.